src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -592,12 +593,17 @@ export default class AircraftModel {
         let assignedAltitude = this.mcp.altitude;
         let flightPlanAltitude = this.fms.flightPlanAltitude;
 
+
         if (assignedAltitude === INVALID_NUMBER) {
             assignedAltitude = '-';
+        } else {
+            assignedAltitude *= UNIT_CONVERSION_CONSTANTS.FT_FL;
```

Comment:
💩 

it seems you know me well, as well.  I see what you're saying, though.  I don't like to see `else`s.

what if, instead, we did this:
```
let assignedAltitude = '-'; 

if (this.mcp.altitude !== INVALID_NUMBER) {
  assignedAltitude = this.mcp.altitude * UNIT_CONVERSION_CONSTANTS.FT_FL;
}
```

I don't like this because we will hit the if more often than not, but its cleaner.